code/style.css

```diff
@@ -1,7 +1,503 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+
+
+* {
+  margin: 0;
+  padding: 0;
+  
+}
+html {
+  height: 100vh;
+  font-size: 18px;
+}
+
+/*** Body ***/
+body {
+  height: 100%;
+  box-sizing: border-box;
```

Comment:
I am not sure but generally its better to use universal selector for box-sizing-